### PR TITLE
Run testDebug builds in place of test in CI

### PR DESCRIPTION
**Pull request by michael-paystack**
Running `gradlew testDebug` allows access to classes in the debug source set

### CODE REVIEW

File: .github/workflows/SonarCloud.yml [hunk 1]
```diff
@@ -32,5 +32,5 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        run: ./gradlew build test sonarqube --scan
+        run: ./gradlew build testDebug sonarqube --scan
```
Comment (Peter-John-paystack) [inline .github/workflows/SonarCloud.yml:35]: What is is the purpose of this change?

Comment (michael-paystack) [reply .github/workflows/SonarCloud.yml:35]: @Peter-John-paystack I'm putting a `TestActivity` in the debug source folder to use for Robolectric tests

Comment (Peter-John-paystack) [reply .github/workflows/SonarCloud.yml:35]: @michael-paystack  But test runs debug variant, correct me if im wrong? Have you run `gradle test` locally to see if it doesnt run your `TestActivity`

Comment (michael-paystack) [reply .github/workflows/SonarCloud.yml:35]: Yea @Peter-John-paystack, I have. It doesn't include the debug source set

**Review by ravi-paystack — APPROVED** [review]

**Review by Peter-John-paystack — APPROVED** [review]

**Pull request merged**